database/ffldb/dbcache.go

```diff
@@ -24,19 +24,6 @@ const (
 	// threshold in between database cache flushes when the cache size has
 	// not been exceeded.
 	defaultFlushSecs = 300 // 5 minutes
-
-	// ldbBatchHeaderSize is the size of a leveldb batch header which
```

Comment:
If you haven't already - Please check where was that used, and maybe we want to use it ourselves?

Reply:
It was never used